packages/core/src/common/keybinding.ts

```diff
@@ -144,27 +151,62 @@ export class KeybindingRegistry {
         }
     }
 
-    registerKeybindings(...bindings: Keybinding[]): void {
+    registerKeybindings(...bindings: (RawKeybinding | Keybinding)[]): void {
         for (const binding of bindings) {
             this.registerKeybinding(binding);
         }
     }
 
+    protected keybindingFromRaw(binding: RawKeybinding): Keybinding | undefined {
+        if (this.commandRegistry.getCommand(binding.command)) {
+            const code = KeyCode.parse(binding.keybinding);
```

Comment:
it probably should throw an error with the concrete parse error message.

Reply:
I don't think it should stop the initialization of the application. Logging an error seems right.

Reply:
I meant `KeyCode.parse` should throw why parsing fails and it can be logged here.

Reply:
Yes I agree I'll add a second commit on top with that change.